src/syntax/softposit.rkt

```diff
@@ -0,0 +1,190 @@
+#lang racket
+
+
+(require math/bigfloat racket/lazy-require)
+(lazy-require [softposit-rkt (_posit8 _posit16 _posit32 _posit64 _posit128
```

Comment:
Ah, I see, the idea here is that it only tried loading the `softposit-rkt` library if you use a `posit` operation, at which point you'll get an error. I think that's fine for now but we'll want some longer-term way to determine if the operations are available.